src/main/java/at/hannibal2/skyhanni/data/ChatManager.kt

```diff
@@ -41,7 +43,7 @@ object ChatManager {
             override fun removeEldestEntry(
                 eldest: MutableMap.MutableEntry<IdentityCharacteristics<IChatComponent>, MessageFilteringResult>?,
             ): Boolean {
-                return size > 100
+                return size > config.chatHistoryLength
```

Comment:
```suggestion
                return size > config.chatHistoryLength.coerceAtLeast(0)
```

kinda useless but since you can put any number into that field, its probably for the better? idk depends on what others think

Reply:
eh it probably doesnt matter actually